internal/cli/cmd/auth/login.go

```diff
@@ -58,8 +59,14 @@ func NewLoginCmd() *cobra.Command {
 				return err
 			}
 
-			if err := auth.StoreTenantToken(tenant.token); err != nil {
-				return err
+			if fnapi.AdminMode {
```

Comment:
There are two login methods implemented in this file:
* the old one (getting a Clerk token and then performing Exchange) 
* and the new one (getting a Tenant token directly).

What you say only applies to the old login method. It will be removed soon. Let's say that for admin access we'll need to use the tenant login method.

For the new one, Exchange will be called within HandoffTenantLogin on the server side. I'll be sure to pass "admin" scope.

FWIW, Niklas added an explicit "impersonate" parameter to Exchange: https://github.com/namespacelabs/internal/commit/14ab32f93406770353167d47f7b33c152b2f3d1d . Technically, we added it because we overlooked the "admin" scope, but I do think that the actual parameter is better than checking the scopes list.